internal/cloudapi/v2/server.go

```diff
@@ -126,6 +126,9 @@ func (s *Server) enqueueCompose(distribution distro.Distro, bp blueprint.Bluepri
 	ir := irs[0]
 
 	ibp := blueprint.Convert(bp)
+	// shortcut
+	arch := ir.imageType.Arch()
```

Comment:
The comment here doesn't really help much.  I think it's clear what's going on from the code.